Gruntfile.coffee

```diff
@@ -127,8 +140,18 @@ module.exports = (grunt) ->
 
 	grunt.registerTask 'prepublish', [
 		'checkDependencies'
-		'clean'
+		'clean:out'
+		'copy:prepublish'
+		'coffee:prepublish'
+		'ts:prepublish'
+	]
+
+	grunt.registerTask 'rebuild', [
```

Comment:
I think you can make this
```coffee
grunt.registerTask 'rebuild', [
	'prepublish'
	'clean:target'
	'copy:target'
]
```
and just inherit the `prepublish` tasks rather than having to copy them